app/ts/components/pages/ChangeChain.tsx

```diff
@@ -72,9 +72,7 @@ export function ChangeChain() {
 								<b> { chainChangeData.rpcNetwork.name } </b>
 							</p>
 							{ chainChangeData.rpcNetwork.httpsRpc === undefined && chainChangeData.simulationMode ?
-								<ErrorContainer
-									text = { 'This chain is not supported by The Interceptor. If you want to use this chain anyway. Select Signing mode instead of Simulation mode and attempt to change the chain again. You will then be able to disable The Interceptor and send transactions without its protection.' }
-								/>
+								<ErrorComponent text = { 'This chain is not supported by The Interceptor. If you want to use this chain anyway. Select Signing mode instead of Simulation mode and attempt to change the chain again. You will then be able to disable The Interceptor and send transactions without its protection.' }/>
```

Comment:
should we use "network" instead of "chain" here since the resulting change prompt from the user's wallet (like MetaMask) use that term?